docs/iris/Makefile

```diff
@@ -7,16 +7,16 @@ html:
 
 pdf:
 	@for i in $(SUBDIRS); do\
-	echo "make latex in $$i.."; \
-	(cd $$i; $(MAKE) $(MFLAGS) $(MYMAKEFLAGS) latex); done
-	echo "\def\sphinxdocclass{MO_report}" > build/latex/docs.tex
-	echo "\documentclass[letterpaper,10pt,english]{MO_report}" >> build/latex/docs.tex
-	tail -n +4 build/latex/Iris.tex >> build/latex/docs.tex
-	sed 's/\\tableofcontents/\\tableofcontents\n\\pagenumbering\{arabic\}/' build/latex/docs.tex > build/latex/docs2.tex
-	sed 's/subsection{/section{/' build/latex/docs2.tex > build/latex/documentation.tex
-	(cd build/latex; pdflatex -interaction=scrollmode documentation.tex)
-	# call latex again to get page numbers right...
-	(cd build/latex; pdflatex -interaction=scrollmode documentation.tex);
+		echo "make latex in $$i.."; \
+		(cd $$i; $(MAKE) $(MFLAGS) $(MYMAKEFLAGS) latex); done
```

Comment:
The `done` on the look is here; the rest shouldn't be indented.

Reply:
Good spot.  The "make pdf" has not been used in a very long time.  As the Read The Docs service can now do this I will remove the "pdf" target.